zio/csvio/writer.go

```diff
@@ -0,0 +1,84 @@
+package csvio
+
+import (
+	"encoding/csv"
+	"errors"
+	"io"
+	"time"
+
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zng"
+	"github.com/brimsec/zq/zng/flattener"
+	"github.com/brimsec/zq/zng/resolver"
+)
+
+var ErrNotDataFrame = errors.New("Cannot format non-dataframe group of records as csv.")
+
+type Writer struct {
+	epochDates bool
+	writer     io.WriteCloser
+	encoder    *csv.Writer
+	flattener  *flattener.Flattener
+	format     zng.OutFmt
+	first      *zng.TypeRecord
+}
+
+func NewWriter(w io.WriteCloser, utf8, epochDates bool) *Writer {
+	var format zng.OutFmt
+	if utf8 {
+		format = zng.OutFormatZeek
+	} else {
+		format = zng.OutFormatZeekAscii
+	}
```

Comment:
```suggestion
	format := zng.OutFormatZeekAscii
	if utf8 {
		format = zng.OutFormatZeek
	}
```

Reply:
This came from textio and tableio.  Fixed there too.  Also, I needed to change const declaration in zng.go to cast to zng.OutFmt, which was needed anyway.